docs/tools/testing/era-test-node.md

```diff
@@ -87,48 +77,45 @@ Please note that the existing implementation does not facilitate communication w
 
 ## Forking networks
 
-To fork the mainnet, use the following command:
+To fork the testnet, use the following command:
 
 ```bash
-# era_test_node fork <NETWORK>
-era_test_node fork mainnet
+era_test_node fork testnet
 ```
 
 :::tip
-You can also fork testnet with `era_test_node fork testnet`
+You can also fork testnet with `era_test_node fork mainnet`
```

Comment:
```suggestion
You can also fork mainnet with `era_test_node fork mainnet`
```

Reply:
This line should actually be testnet testnet considering the rest of the file continues reading about testnet